src/vs/workbench/services/activity/common/activityService.ts

```diff
@@ -70,4 +71,19 @@ export interface IActivityService {
 	 * Clears activity shown in the activitybar for the given viewlet or panel.
 	 */
 	clearActivity(compositeId: string): void;
-}
\ No newline at end of file
+
```

Comment:
@octref this Viewlet related code should move into `IViewletService` or if that is impossible for some reason maybe into a new `IExtensionViewletService`